nexus/src/db/model.rs

```diff
@@ -631,16 +631,48 @@ pub struct Sled {
     pub ip: ipnetwork::IpNetwork,
     // TODO: Make use of SqlU16
     pub port: i32,
+
+    /// The last IP address provided to an Oxide service on this sled
+    pub last_used_address: IpNetwork,
 }
 
+// TODO-correctness: We need a small offset here, while services and
+// their addresses are still hardcoded in the mock RSS config file at
+// `./smf/sled-agent/config-rss.toml`. This avoids conflicts with those
+// addresses, but should be removed when they are entirely under the
+// control of Nexus or RSS.
```

Comment:
Good call, I'll add a reference. We might always have a small offset, at least partially because the space is so honking big. The current plan in RFD 259 is for RSS to populate the initial addresses into the database, which might mean that it goes through something like the current bump-allocation path, obviating the need for an offset. We'll have to see what shakes out, but I'll definitely refer to #732 in the mean time!